### PR TITLE
Update Web App guidelines

### PR DESCRIPTION
**Pull request by at88mph**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -97,7 +97,44 @@ Once a patch has been merged into the default branch it can, depending on the ur
 
 ### Web Browser Application Development Guidelines
 
-TODO
+Our web applications span many years and are made up of many common technologies such as Vanilla JavaScript, jQuery, CSS, and HTML, but we've also, in recent years, adopted ReactJS in our [Science Portal](https://github.com/opencadc/science-portal) application.
```
Comment (brianmajor) [inline CONTRIBUTING.md]:
```suggestion
Our web applications span many years and are made up of many common technologies such as vanilla JavaScript, jQuery, CSS, and HTML, but we've also, in recent years, adopted ReactJS in our [Science Portal](https://github.com/opencadc/science-portal) application.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -97,7 +97,44 @@ Once a patch has been merged into the default branch it can, depending on the ur
...
+
+#### Coding conventions
+
+We optimize for readability, and while not fully enforced at build time, CheckStyle is used in reviews and ensuring consistent style in the Java code.  See the [cadc-quality](https://github.com/opencadc/core/tree/master/cadc-quality) OpenCADC library for more information.  You can add the [cadc_checkstyle.xml](https://raw.githubusercontent.com/opencadc/core/master/cadc-quality/src/main/resources/cadc_checkstyle.xml) to your IDE's CheckStyle configuration to check as you type.
+
+##### JavaScript
+
+- No semi-colons at end of statement
+- Be consistent with quotes (i.e. pick single or double, and stick with it, or be consistent with the rest of the file)
+- Use Vanilla JavaScript when available
```
Comment (brianmajor) [inline CONTRIBUTING.md]:
```suggestion
- Use vanilla JavaScript when available
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -97,7 +97,44 @@ Once a patch has been merged into the default branch it can, depending on the ur
...
+
+Send a GitHub Pull Request to the appropriate [Repository](#web-browser-application-development-guidelines).  Ensure a Git Message is submitted with commits, and keep them small to ensure an accurate Code Review.
+
+#### Coding conventions
+
+We optimize for readability, and while not fully enforced at build time, CheckStyle is used in reviews and ensuring consistent style in the Java code.  See the [cadc-quality](https://github.com/opencadc/core/tree/master/cadc-quality) OpenCADC library for more information.  You can add the [cadc_checkstyle.xml](https://raw.githubusercontent.com/opencadc/core/master/cadc-quality/src/main/resources/cadc_checkstyle.xml) to your IDE's CheckStyle configuration to check as you type.
+
+##### JavaScript
+
+- No semi-colons at end of statement
```
Comment (brianmajor) [inline CONTRIBUTING.md]:
```suggestion
- No semicolons at end of statement
```

**Review by brianmajor — CHANGES_REQUESTED** [review]
Looks great Dustin, just a few spelling suggestions.

**Comment by at88mph** [comment]
Changes made.

**Review by brianmajor — APPROVED** [review]

**Pull request merged**